admin_manual/installation/source_installation.rst

```diff
@@ -22,20 +22,33 @@ Apache and MariaDB, using `the Nextcloud .tar archive
 * :ref:`php_fpm_tips_label`
 * :ref:`other_HTTP_servers_label`
 
-.. note:: Admins of SELinux-enabled distributions such as CentOS, Fedora, and 
-   Red Hat Enterprise Linux may need to set new rules to enable installing 
+.. note:: Admins of SELinux-enabled distributions such as CentOS, Fedora, and
+   Red Hat Enterprise Linux may need to set new rules to enable installing
    Nextcloud. See :ref:`selinux_tips_label` for a suggested configuration.
 
+.. _snaps_label:
+
+Installing via Snap Packages
+-------------------------------------
+
+A snap is a zip file containing an application together with its dependencies, and a description of how it should safely be run on your system, especially the different ways it should talk to other software. Most importantly snaps are designed to be secure, sandboxed, containerised applications isolated from the underlying system and from other applications. 
+    
+To install the Nextcloud Snap Package, run the following command in a terminal::
+
+    sudo snap install nextcloud
+
+.. note:: The `snapd technology <http://snapcraft.io/docs/core/>`_ is the core that powers snaps, and it offers a new way to package, distribute, update and run OS components and applications on a Linux system. See more about snaps on `snapcraft.io <http://snapcraft.io/>`_.
+
 .. _prerequisites_label:
 
-Prerequisites
--------------
+Prerequisites for Manual Installation
+---------------------------------------------
 
-The Nextcloud .tar archive contains all of the required PHP modules. This 
-section lists all required and optional PHP modules.  Consult the `PHP manual 
-<http://php.net/manual/en/extensions.php>`_ for more information on modules. 
-Your Linux distribution should have packages for all required modules. You can 
-check the presence of a module by typing ``php -m | grep -i <module_name>``. 
+The Nextcloud .tar archive contains all of the required PHP modules. This
```

Comment:
This is not clear, when all modules are contained in the .tar archive, why do I need a list of required and optional modules? Does the tar-archive contain any php module?